tests/build-docs/buildNavTree.test.js

```diff
@@ -0,0 +1,106 @@
+const { buildNavTree } = require('../../scripts/build-docs');
+const { basicNavItems, sectionNavItems, orphanNavItems, missingSpecVersion, nullNavItems } = require('../fixtures/buildNavTreeData')
+
+jest.mock('lodash/sortBy', () => jest.fn(arr => arr));
```

Comment:
I don't think there's a need to mock lodash sortBy. Let it run and expect the results to be sorted, as they should be with the real data.